Makefile-libostree.am

```diff
@@ -54,7 +54,7 @@ src/libostree/ostree-enumtypes.h: src/libostree/ostree-enumtypes.h.template $(EN
 	--template $< \
 	$(ENUM_TYPES) > $@.tmp && mv $@.tmp $@
 
-src/libostree/ostree-enumtypes.c: src/libostree/ostree-enumtypes.c.template $(ENUM_TYPES)
+src/libostree/ostree-enumtypes.c: src/libostree/ostree-enumtypes.c.template src/libostree/ostree-enumtypes.h $(ENUM_TYPES)
 	$(AM_V_GEN) $(GLIB_MKENUMS) \
 	--template $< \
 	$(ENUM_TYPES) > $@.tmp && mv $@.tmp $@
```

Comment:
Note: We could also change this to use `--output $@` to avoid the `.tmp`-and-`mv` dance. The full recommended rules (from the [new documentation](https://bugzilla.gnome.org/show_bug.cgi?id=788948)) are:
```
# A list of headers to inspect
project_headers = \
        project-foo.h \
        project-bar.h \
        project-baz.h

enum-types.h: $(project_headers) enum-types.h.in
        $(AM_V_GEN)$(GLIB_MKENUMS) \
                --template=enum-types.h.in \
                --output=$@ \
                $(project_headers)

enum-types.c: $(project_headers) enum-types.c.in enum-types.h
        $(AM_V_GEN)$(GLIB_MKENUMS) \
                --template=enum-types.c.in \
                --output=$@ \
                $(project_headers)

BUILT_SOURCES += enum-types.h enum-types.c
CLEANFILES += enum-types.h enum-types.c
EXTRA_DIST += enum-types.h.in enum-types.c.in
```

Reply:
If it's important to you, I can make this change, but it will only delay getting the ref binding stuff you need landed :)

Reply:
I’ll do it as a follow-up.

Reply:
Done: https://github.com/ostreedev/ostree/pull/1329